src/RA_Core.cpp

```diff
@@ -68,7 +81,8 @@ static void InitCommon(HWND hMainHWND, /*enum EmulatorID*/int nEmulatorID, const
 {
     // initialize global state
     g_EmulatorID = static_cast<EmulatorID>(nEmulatorID);
-    g_RAMainWnd = hMainHWND;
+    if (!g_RAMainWnd)
+        g_RAMainWnd.reset(hMainHWND);
```

Comment:
So, if for whatever reason, the client calls Init twice, we'll destroy the window associated to the first handle they passed us? This is just a reference to a window owned by the client. We shouldn't be calling `DestroyWindow` on something we didn't create.

Reply:
makes sense.